cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
```

Comment:
This is just the boilerplate Cypress file, so that we have a fixtures directory ready for use later on.

I don't know anything about using fixtures, but I think we might want to "fake" some comparator responses later on?

Reply:
Yeah, definitely useful to get some fixtures here in the future.